client/public/Marker.css

```diff
@@ -0,0 +1,30 @@
+.leaflet-marker-icon {
+	border: 0cm;
+	background: rgba(0, 0, 0, 0);
+}
+
+.marker-text {
+	font-size: 9px !important;

```

Comment:
is this `!important` important? if possible, can we do away with all of the importants?

Reply:
Some of them are important as it gets overriden by existing Chakra styles, but I removed the unnecessary ones